auslib/web/public/api.py

```diff
@@ -0,0 +1,157 @@
+import urllib
```

Comment:
Naming quibble: I don't think "api.py" is the right name for this - we should save that for when there's a real public API (eg: one with endpoints like /releases and /rules). I think we should stick with "client.py" for this, because they are endpoints that exist to serve the update client in Firefox.